fhirpipe/utils.py

```diff
@@ -0,0 +1,32 @@
+def build_col_name(table, column, owner=None):
+    return "{}{}.{}".format(owner + "." if owner is not None else "", table, column,)
+
+
+def new_col_name(script_name, init_col):
+    if isinstance(init_col, tuple):
+        return script_name + "_" + "_".join(init_col[0]) + "_" + "_".join(init_col[1])
+    return "_".join((script_name, init_col))
```

Comment:
Maybe better explain what this function does

and you can use f-string, they are faster and more readable:
```python 
    if isinstance(init_col, tuple):
        return f"{script_name}_{'_'.join(init_col[0])}_{'_'.join(init_col[1])}"
    return f"{script_name}_{init_col}"
```